modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/DataSet.js

```diff
@@ -338,7 +336,9 @@ const DataSet = ({
 	};
 
 	useEffect(() => {
-		setDataLoading(true);
+		if (!apiURL) {
+			return;
+		}
```

Comment:
We still need `setDataLoading(true);`, after `if` check.

Reply:
Done! Thanks @markocikos